cluster/cluster_test.go

```diff
@@ -83,7 +83,6 @@ func TestEncode(t *testing.T) {
 		require.NoError(t, err)
 
 		require.Equal(t, b1, b2)
-		require.Equal(t, definition, definition2)
```

Comment:
if raw bytes are equal then it doesn't make sense for this require

Reply:
rather add an ``Equal` method to Definition